assets/stylesheets/bootstrap/_panels.scss

```diff
@@ -5,23 +5,37 @@
 
 // Base class
 .panel {
-  margin-bottom: $line-height-computed;
+  margin-top: 0.5rem;
   background-color: $panel-bg;
-  border: 1px solid transparent;
-  border-radius: $panel-border-radius;
+  border-left: 3px solid $panel-primary-border;
+  border-bottom: 1px solid $panel-footer-bg;
+
+  &.odd {
+    background-color: $white;
+  }
+
+  .flexbox & {
+    & .panel-body {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+    }
+  }
 }
 
 // Panel contents
 .panel-body {
+  color: $black;
```

Comment:
Whilst I'm here, we probably need something like `.panel-nested` and use that class to remove the border top as it's too thick.